src/Data/Recruit.ChangeScripts/050-vacancyReviews_migration.js

```diff
@@ -0,0 +1,7 @@
+{
+    print((new Date()) + "Start migrating documents from reviews to vacancyReviews collection.");
```

Comment:
You need a space before `Start` and same on `Finish` line.

Could you also print the document count on source collection before migration and after.